src/router/index.ts

```diff
@@ -122,27 +122,27 @@ router.get("/experiences/:id/votes", async (req: Request, res: Response) => {
     const { id } = req.params;
 
     if (!id) {
-      return res.send({ message: "ID must be provided!" });
+      res.send({ message: "ID must be provided!" });
     }
 
-    // const parsedId = parseInt(id, 10);
+    const parsedId = parseInt(id, 10);
 
-    // // const votes = await prisma.votes.findMany({
-    // //   where: { experience_id: parsedId },
-    // //   include: {
-    // //     users_accounts: true,
-    // //   },
-    // // });
+    const votes = await prisma.votes.findMany({
+      where: { experience_id: parsedId },
```

Comment:
Votes are related to place_etiquettes, they don't have anything to do with an experience.